tools/monitoring/alert/alert.go

```diff
@@ -67,11 +63,14 @@ func (m *MailConfig) Alert(errorPattern string, config *config.SelectedConfig, d
 		return false, err
 	}
 
-	err = m.sendAlert(errorPattern, config)
+	content := mailContent{*report, errorPattern, s.Hint, s.Duration()}
+	err = m.sendAlert(&content)
 	return err == nil, err
 }
 
 // checkAlertsTable checks alert table and see if it is necessary to send alert email
+// returns a sql statement that contains db update action to perform. An empty string indicates
+// no db operations to perform and no alert email needs to be sent
```

Comment:
out-dated comment?

Reply:
right. It's out of date. It's auto-merged and I didn't notice it